app/migration/migrators/school.rb

```diff
@@ -25,12 +25,14 @@ def self.record_count = schools.count
     def self.reset! = nil
 
     def self.schools
-      ::Migration::School.with(eligible_or_cip_or_with_irs:
+      ::Migration::School.with(eligible_or_cip_or_with_irs_or_partnerships:
```

Comment:
`irs` made me think of the [Inland Revenue Service](https://en.wikipedia.org/wiki/Inland_Revenue) 😂